hivesim/doc.go

```diff
@@ -1,3 +1,126 @@
-// Package hivesim is a Go wrapper for the Hive Simulation API.
-// You can use this package to write simulations for Hive in Go.
+/*
+	Package hivesim is a Go wrapper for the Hive Simulation API.
```

Comment:
We might, in the README, point to godocs for additional package docs. Just to make it easier for people to find the package-specific documentation, which until now, maybe we've been kind of bad at maintaining

Reply:
@holiman I added a `godocs` button to the README.md, do you think that's sufficient?